src/main/java/org/codeforamerica/shiba/pages/PageUtils.java

```diff
@@ -46,4 +49,16 @@ public static String getTitleString(List<String> strings) {
 
         return Stream.concat(applicant, nonApplicantHouseholdMembers).collect(Collectors.toList());
     }
+
+    public static Boolean isCCAPEligible(DatasourcePages datasourcePages) {
+        List<String> applicantPrograms = datasourcePages.get("choosePrograms").get("programs").getValue();
+        boolean applicantHasCCAP = applicantPrograms.contains("CCAP");
+        boolean hasHousehold = !datasourcePages.get("householdMemberInfo").isEmpty();
+        boolean householdHasCCAP = false;
+        if (hasHousehold) {
+            householdHasCCAP = datasourcePages.get("householdMemberInfo").get("programs").getValue().stream().anyMatch(iteration ->
+                    iteration.contains("CCAP"));
+        }
+        return applicantHasCCAP || householdHasCCAP;
+    }
```

Comment:
We're under time constraints so it's nbd for now but we should find a way to refactor this to use the DocumentListParser so that we only have this logic in one place.